app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -685,9 +687,17 @@ open class HomeActivity : LocaleAwareAppCompatActivity(), NavHostActivity {
         return super.onKeyDown(keyCode, event)
     }
 
-    final override fun onKeyUp(keyCode: Int, event: KeyEvent?): Boolean {
+    final override fun onKeyUp(keyCode: Int, event: KeyEvent): Boolean {
         if (shouldUseCustomBackLongPress() && keyCode == KeyEvent.KEYCODE_BACK) {
             backLongPressJob?.cancel()
+
+            // check if the tab history if already visible.
+            if (event.eventTime - event.downTime >= ViewConfiguration.getLongPressTimeout() &&
```

Comment:
drive by: would it be possible to break out the `event.eventTime - event.downTime >= ViewConfiguration.getLongPressTimeout()` into a local `val` with a clear name of what the result is? Having a temporary variable to store the result may help with the readability.

Reply:
I added a longer explanation in the comment, about what exactly are we checking for heree.